pkg/diagnose/runner.go

```diff
@@ -407,45 +435,88 @@ func runStdOut(w io.Writer, diagCfg diagnosis.Config, run func(diagnosis.Config)
 		color.NoColor = true
 	}
 
-	fmt.Fprintf(w, "=== Starting diagnose ===\n")
+	if !diagCfg.JSONOutput {
+		fmt.Fprintf(w, "=== Starting diagnose ===\n")
+	}
 
 	diagnoses, err := run(diagCfg)
-	if err != nil && !diagCfg.RunLocal {
-		fmt.Fprintln(w, color.YellowString(fmt.Sprintf("Error running diagnose in Agent process: %s", err)))
-		fmt.Fprintln(w, "Running diagnose command locally (may take extra time to run checks locally) ...")
-
-		// attempt to do so locally
+	if err != nil {
```

Comment:
Can you explain the removal of `!diagCfg.RunLocal`

Reply:
it's something I already fixed in a commit that is not pushed yet, but it's coming